cmd/odf/get/health.go

```diff
@@ -0,0 +1,24 @@
+package get
+
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/health"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/spf13/cobra"
+)
+
+var clusterHealth = &cobra.Command{
+	Use:                "health",
+	Short:              "check health of the cluster and common configuration issues",
+	DisableFlagParsing: true,
+	Hidden:             true,
```

Comment:
We don't want to hide this, right?

Reply:
hmm we can skip this to hide as this is read-only operation I kept this Eran comment was on overall Epic. But I'll update him about this

Reply:
Yeah seems like no need to hide read only operations